src/version.js

```diff
@@ -4,3 +4,4 @@ export const VERSION = React.version;
 export const REACT013 = VERSION.slice(0, 4) === '0.13';
 export const REACT014 = VERSION.slice(0, 4) === '0.14';
 export const REACT15 = VERSION.slice(0, 3) === '15.';
+export const REACT155 = VERSION.slice(0, 4) === '15.5';
```

Comment:
maybe `= REACT15 && VERSION.slice(3, 5) === '5.';` would be more accurate? I don't want this matching `15.50` one day ;-)